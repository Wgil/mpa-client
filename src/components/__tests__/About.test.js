import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import About from '../About';

describe('Test About.js', () => {
  it('Component Rendered', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toBeTruthy();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
