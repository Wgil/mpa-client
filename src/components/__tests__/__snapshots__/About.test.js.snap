// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test About.js Component Rendered 1`] = `
<section>
  <HeaderLite />
  <div
    className="App"
  >
    <header
      className="App-header pt-2 p-5"
    >
      <h1
        className="App-title text-white pt-5"
      >
        About the Missing Person's Project
      </h1>
      <p
        className="pl-md-5 pr-md-5"
      >
        This platform will maintain an archive of all information which is published within its domain, users can query the database to search for current cases, un-resolved and resolved cases, and see if there is a law enforcement handler on any of the cases.
      </p>
    </header>
  </div>
  <Container
    tag="div"
  >
    <div
      className="mt-5 mb-4 d-flex flex-wrap flex-row offset-md-1 text-center"
    >
      <PreFooter
        alt="learn how we work"
        heading="Who we Are"
        image="document.svg"
      />
      <PreFooter
        alt="report a case"
        heading="Report a Sighting"
        image="report.svg"
      />
      <PreFooter
        alt="donate and contribute"
        heading="Support Us"
        image="hands.svg"
      />
    </div>
  </Container>
  <Footer />
</section>
`;
